episodes/pathology.md

```diff
@@ -0,0 +1,200 @@
+---
+title: "Working with Pathology Images"
+teaching: 60
+exercises: 30
+---
+
+:::::::::::::::::::::::::::::::::::::: questions 
+
+- What are some open libraries can be used with pathology?
+- How are pathology slides different from a photo?
+- What is the structure of a pathology slide?
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: objectives
+
+- Describe some open source tools and libraries for pathology images
+- Discuss  common file formats for pathology slides
+- Explain tiling and pyramidal file formats
+- Show basic code in openslide
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+## Introduction
+
+Pathologists deal with multiple types of images. Some pathologists may use photography when recording gross (or macroscopic) pathology data. The processing of digital photography is a straightforward application of general image processing. However the workhorse type image of pathology is the histopathological image, and this is very different from a photograph. 
+
+In the below picture you can see an image with gross and histopathology of signet ring cell carcinoma metastasis to the ovary. The histopathology images are stained microscopy image. The gross pathology images are photographs.
+
+<img src="fig/path_metastasis.jpg" alt="Gross and histopathology" width="70%;"/>
+
+*Nakamura, Yoshiaki; Hiramatsu, Ayako; Koyama, Takafumi; Oyama, Yu; Tanaka, Ayuko; Honma, Koichi, CC BY 3.0 <https://creativecommons.org/licenses/by/3.0>, via Wikimedia Commons*
+
+In this lesson we will cover how to work with histopathology images as they differ from simpler multi-channel color image like a modern digital camera often produces. 
+
+## File formats
+
+In terms of histopathological images, you may find them in several file formats. Not every library or tool for such imaging works with every format. Below is a table with several file formats.
+
+
+| Format Name |Extension|Notes|  More info|
+| ----------- |---------|-----|-----------|
+| TIFF     |.tiff|tag image file format| can be tiled |https://www.loc.gov/preservation/digital/formats/fdd/fdd000022.shtml|
+| OME-TIFF       |.ome.tiff, ome.tif, .ome.tf2, .ome.tf8 or .ome.btf | includes XML and may be multiple TIFFs|https://brainder.org/2012/09/23/the-nifti-file-format/|
+| SVS         | .svs    | Scanned Virtual Slide  |https://www.dicomstandard.org/  |
+| DICOM       |.dcm or none|supports an included WSI file|https://dicom.nema.org/dicom/dicomwsi/|
```

Comment:
none if part of a DICOM media..

Reply:
I just meant there are a lot of dicom files that just have no file extension on them (at least no visible file extension and it drives me crazy).

Reply:
yes, this is clear.